<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadPage</class>
 <widget class="QWizardPage" name="DownloadPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <property name="spacing">
    <number>15</number>
   </property>
   <item>
    <widget class="QLabel" name="filename_label_">
     <property name="styleSheet">
      <string notr="true">font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Downloading:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progress_bar_">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="details_group_">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="details_layout">
      <item>
       <widget class="QLabel" name="progress_label_">
        <property name="text">
         <string>Downloaded: 0 MB / 0 MB</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="speed_label_">
        <property name="text">
         <string>Speed: -- MB/s</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="remaining_label_">
        <property name="text">
         <string>Remaining: --</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="dest_title_">
     <property name="styleSheet">
      <string notr="true">font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Destination:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="destination_label_">
     <property name="styleSheet">
      <string notr="true">color: #666;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="status_group_">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="status_layout">
      <item>
       <widget class="QCheckBox" name="deps_checkbox_">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Installing Python dependencies...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="verify_checkbox_">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Verifying model checksum...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="test_checkbox_">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Testing model loading...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cancel_button_">
     <property name="text">
      <string>Cancel Download</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
